### ISSUE TITLE
Communication Score < 10k - can't ping own abilities

### ISSUE THREAD

**Issue by ADMarian** [ValveSoftware/Dota2-Gameplay#18820]
### Description

If your communication score is less than 10k you can't ping your own abilities. This makes is very annoying that you can't ping your smoke to let your teammates know you're ready to go, or if your ult is own cd etc. 

I think it should prevent you from pinging ally abilities and stuff not yours!

**Comment by Neronuser** [comment]
Agreed, I play Enigma a lot, communication score fluctuating around 9.9-10k, and I can't ping my ult cooldown to let the team know I am not ready for a big team fight. It is very frustrating
